students/grzegorz_bajorski/lesson_02_flow_control/factorial.py

```diff
@@ -0,0 +1,7 @@
+n = int(input())
+
+for i in range(1,n):
+    n *= i
+
+print(n)
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.